igor-web/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -71,7 +71,8 @@
 
 public class TravisService implements BuildOperations, BuildProperties {
 
-  static final int TRAVIS_BUILD_RESULT_LIMIT = 100;
+  static final int TRAVIS_JOB_RESULT_LIMIT = 100;
+  static final int TRAVIS_BUILD_RESULT_LIMIT = 10;
```

Comment:
will it make more sense to have it configured in igor.yml instead of hard coded?

Reply:
The second one could maybe make sense to put in a config file. The first one is the max number of results per page, so that is ok to hard code.

Reply:
Yep, I think leaving the first one here it's fine, but if you can move the other param to be configurable it would be great.